frontend/src/metabase/query_builder/actions/sharing.ts

```diff
@@ -29,19 +27,23 @@ export const deletePublicLink = createThunkAction(
     await dispatch(deleteCardPublicLink.initiate(card)),
 );
 
-export const UPDATE_ENABLE_EMBEDDING = "metabase/card/UPDATE_ENABLE_EMBEDDING";
-export const updateEnableEmbedding = createAction(
-  UPDATE_ENABLE_EMBEDDING,
-  ({ id }: CardIdPayload, enable_embedding: boolean) =>
-    CardApi.update({
-      id,
-      enable_embedding,
-    }),
-);
-
-export const UPDATE_EMBEDDING_PARAMS = "metabase/card/UPDATE_EMBEDDING_PARAMS";
-export const updateEmbeddingParams = createAction(
-  UPDATE_EMBEDDING_PARAMS,
-  ({ id }: CardIdPayload, embedding_params: EmbedOptions) =>
-    CardApi.update({ id, embedding_params }),
+export const updateEnableEmbedding = createAsyncThunk<
+  Pick<Card, "id" | "enable_embedding"> & { uuid: Card["public_uuid"] },
```

Comment:
We can probably improve this one. Should we just make a new type?

Reply:
Making a new type sounds good

Reply:
actually.. I'll leave this. I'll make the type when I migrate the embedding endpoints to RTK in [this PR](https://github.com/metabase/metabase/pull/48391)